File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P07_HotelRoom.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	string month;
	int nights;
	cin >> month >> nights;

	double studioPrice = 0.0;
	double appartmentPrice = 0.0;
	if (month == "May" || month == "October") {
		studioPrice = 50;
		appartmentPrice = 65;

		if (7 < nights && nights <= 14) {
			studioPrice *= 0.95;
		} else if (14 < nights) {
			studioPrice *= 0.7;
			appartmentPrice *= 0.9;
		}
	} else if (month == "June" || month == "September") {
		studioPrice = 75.2;
		appartmentPrice = 68.7;

		if (14 < nights) {
			studioPrice *= 0.8;
			appartmentPrice *= 0.9;
		}
	} else if (month == "July" || month == "August") {
		studioPrice = 76;
		appartmentPrice = 77;

		if (14 < nights) {
			appartmentPrice *= 0.9;
		}
	}

	cout.setf(ios::fixed);
	cout.precision(2);

	cout << "Apartment: " << appartmentPrice * nights << " lv." << endl;
	cout << "Studio: " << studioPrice * nights << " lv." << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P07_HotelRoom.py
month = input()
days = int(input())

cost_a = 0
cost_s = 0

if month == "May" or month == "October":
    cost_a = days * 65
    cost_s = days * 50
    if 7 < days <= 14:
        cost_s = cost_s * 0.95
    if days > 14:
        cost_s = cost_s * 0.7
elif month == "June" or month == "September":
    cost_a = days * 68.70
    cost_s = days * 75.20
    if days > 14:
        cost_s = cost_s * 0.8
elif month == "July" or month == "August":
    cost_a = days * 77
    cost_s = days * 76

if days > 14:
    cost_a = cost_a * 0.9

print(f"Apartment: {cost_a:.2f} lv.")
print(f"Studio: {cost_s:.2f} lv.")